ts/src/renderer/phaser/classes/PhaserUnit.ts

```diff
@@ -18,6 +19,8 @@ class PhaserUnit extends PhaserAnimatedEntity {
 			translate.y,
 			[ this.sprite ]
 		);
+		/*const attributesContainer = this.attributesContainer = scene.add.container();
```

Comment:
this should be uncommented in regards with comment below, and set proper y offset